isolation/_02_RepeatableRead/foreign_key_column/answer/delete_truncate_dont_reuse_oid_02.answer

```diff
@@ -0,0 +1,50 @@
+| 2 rows affected
```

Comment:
It seems that this is the same as `delete_truncate_02.answer`. This should be the same as the original answer of it.

Reply:
This result is the same as the actual test result. So it would be good to check if it is a bug. 
Also with `isolation/_02_RepeatableRead/foreign_key_column/answer/truncate_delete_dont_reuse_oid_01.answer`

Reply:
It turns out that I misunderstood the case. It is expected result and you should remove `delete_truncate_dont_reuse_oid_02.answer/ctl`.

Reply:
What's wrong with `truncate_delete_dont_reuse_oid_01.answer`?

Reply:
Because `truncate_delete_dont_reuse_oid_01.answer` and `truncate delete 01.answer` are the same.
Should I also delete `truncate_delete_dont_reuse_oid_01.answer` ?

Reply:
Oh, I checked the different file `delete_truncate_01`. You're right. That should be deleted too.